crates/fuel-core/src/graphql_api/ports.rs

```diff
@@ -130,3 +149,38 @@ pub trait DatabaseChain {
 
     fn base_chain_height(&self) -> StorageResult<DaBlockHeight>;
 }
+
+pub trait FindTx {
+    fn find_one(&self, id: TxId) -> Option<TxInfo>;
+}
+
+pub trait InsertTx {
+    fn insert(&self, txs: Vec<Arc<Transaction>>) -> Vec<anyhow::Result<InsertionResult>>;
+}
+
+pub trait TxSubscription {
+    fn tx_update_subscribe(&self) -> tokio::sync::broadcast::Receiver<TxUpdate>;
```

Comment:
Instead of `Receiver` please use `fuel_core_services::stream::BoxStream`=)

Reply:
Changed